File: 01/001/main.go
package main

import "fmt"

func Sum35(max int) int {
	var sum = 0
	for i := 0; i < max; i++ {
		if i%3 == 0 || i%5 == 0 {
			sum += i
		}
	}
	return sum
}

func main() {
	fmt.Println(Sum35(1000))
}

File: 01/001/Main.java
import java.util.stream.IntStream;

public class Main {

    public static int sum35(int max) {
        return IntStream.range(0, max)
            .filter(a -> a % 3 == 0 || a % 5 == 0)
            .sum();
    }

    public static void main(String[] args) {
        System.out.println(sum35(1000));
    }
}
